src/widget/form/genericchatform.cpp

```diff
@@ -466,21 +464,17 @@ void GenericChatForm::onSelectAllClicked()
     chatWidget->selectAll();
 }
 
-QString GenericChatForm::resolveToxId(const ToxId &id)
+QString GenericChatForm::resolveToxId(const ToxId &id) const
 {
     Friend *f = FriendList::findFriend(id);
     if (f)
-    {
         return f->getDisplayedName();
-    }
-    else
+
+    for (auto it : GroupList::getAllGroups())
```

Comment:
Probably should become `for (const auto&& group : GroupList::getAllGroups())`. Btw. `auto` is actually a copy of a group, not an iterator as the variable name suggests.